src/components/global/navigation.astro

```diff
@@ -14,14 +18,15 @@
       />
     </a>
     <div id="header-links" class="flex-row gap-3 z-20 text-sm hidden lg:flex">
-      <a href="/#about" class="font-normal">About</a>
-      <a href="/#news" class="font-normal">News</a>
-      <a href="/#gi" class="font-normal">Get Involved</a>
-      <a href="/#sponsors" class="font-normal">Sponsors</a>
-      <a href="/#faq" class="font-normal">FAQ</a>
+      <a href="/#about" class="font-normal hover:brightness-75">About</a>
+      <a href="/#news" class="font-normal hover:brightness-75">News</a>
+      <a href="/#gi" class="font-normal hover:brightness-75">Get Involved</a>
+      <a href="/#sponsors" class="font-normal hover:brightness-75">Sponsors</a>
+      <a href="/#faq" class="font-normal hover:brightness-75">FAQ</a>
     </div>
     <div id="header-donate" class="hidden lg:block">
       <a href="https://donorbox.org/ladybird" class="btn btn--small">
+        <Icon name="mdi:donation-outline" class="size-5"/>
```

Comment:
I'm not sold on this icon. @gmta ?

<img width="254" alt="Screenshot 2024-12-03 at 10 29 10 PM" src="https://github.com/user-attachments/assets/31a39bac-0170-4b6e-9ae5-a42904a85ac3">

Reply:
I also agree that there might be better alternatives, do you have any suggestions you may like from [Iconify](https://icon-sets.iconify.design/)? I have tried to pick the most fitting one from the MDI icon library, however there may be better options indeed either from MDI or different icon libraries found on Iconify.